### PR TITLE
Added ListAccessibleCustomers example

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/AccountManagement/ListAccessibleCustomers.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\GoogleAdsClient;
+use Google\Ads\GoogleAds\Lib\GoogleAdsClientBuilder;
+use Google\Ads\GoogleAds\Lib\GoogleAdsException;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
+use Google\Ads\GoogleAds\V0\Enums\ChangeStatusResourceTypeEnum\ChangeStatusResourceType;
```
Comment (jimper) [inline examples/AccountManagement/ListAccessibleCustomers.php]: is this used?

File: examples/AccountManagement/ListAccessibleCustomers.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+use Google\Ads\GoogleAds\V0\Errors\GoogleAdsError;
+use Google\Ads\GoogleAds\V0\Resources\ChangeStatus;
```
Comment (jimper) [inline examples/AccountManagement/ListAccessibleCustomers.php]: same here

File: examples/AccountManagement/ListAccessibleCustomers.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient)
+    {
+        $customerServiceClient = $googleAdsClient->getCustomerServiceClient();
+
+        // Issues a request for listing all accessible customers.
+        $accessibleCustomers = $customerServiceClient->listAccessibleCustomers();
+        print 'Total results: ' . count($accessibleCustomers) . PHP_EOL;
+
+        // Iterates over all accessible customers' resource names and print them.
```
Comment (jimper) [inline examples/AccountManagement/ListAccessibleCustomers.php]: "and print" -> "and prints"

File: examples/AccountManagement/ListAccessibleCustomers.php (hunk 1 above)
Comment (fiboknacky) [reply examples/AccountManagement/ListAccessibleCustomers.php]: Removed.

File: examples/AccountManagement/ListAccessibleCustomers.php (hunk 2 above)
Comment (fiboknacky) [reply examples/AccountManagement/ListAccessibleCustomers.php]: Done.

File: examples/AccountManagement/ListAccessibleCustomers.php (hunk 3 above)
Comment (fiboknacky) [reply examples/AccountManagement/ListAccessibleCustomers.php]: Done.

**Review by jimper — APPROVED** [review]
LGTM

**Review by sundquist — APPROVED** [review]
LGTM

**Pull request merged**